src/ipahealthcheck/meta/core.py

```diff
@@ -25,31 +26,30 @@ class MetaCheck(Plugin):
     def check(self):
 
         rval = constants.SUCCESS
-        if not os.path.exists(paths.FIPS_MODE_SETUP):
-            fips = "missing {}".format(paths.FIPS_MODE_SETUP)
-            logger.debug('%s is not installed, skipping',
-                         paths.FIPS_MODE_SETUP)
+        if not os.path.exists(paths.PROC_FIPS_ENABLED):
+            fips = "missing {}".format(paths.PROC_FIPS_ENABLED)
+            logger.debug("Can't find %s, skipping" %
+                         paths.PROC_FIPS_ENABLED)
```

Comment:
I'm a bit torn on this related to other distributions, whether enforcing that the file exist would break them.
When I originally wrote this I didn't want to hold it against a user that they didn't have a tool installed. But the file in /proc should be created by the kernel so if that's missing it points to a larger issue.
I think we should set rval to WARNING in this case. This file is created by the kernel so if FIPS is disabled in the KERNEL that seems odd. A user can suppress the warning if indeed they have this use-case.

Reply:
I think we can make the assumption, that this file exists, otherwise, the user is using a custom kernel and is probably not thinking about FIPS at all. Nevertheless, the return value is a warning now.